### PR TITLE
Add short scroll view

### PR DESCRIPTION
**Pull request by uarlouski**

### CODE REVIEW

**Review by valfirst — APPROVED** [review]

File: src/App.js [hunk 1]
```diff
@@ -38,6 +39,10 @@ const App = () => {
           <Drawer.Screen name="Wait" component={WaitScreen} />
           <Drawer.Screen name="Date Picker" component={DatePickerScreen} />
           <Drawer.Screen name="Scroll View" component={ScrollViewScreen} />
+          <Drawer.Screen
```
Comment (ikalinin1) [inline src/App.js:42]: why it is not in a single line like all the others?

File: src/screens/ShortScrollViewScreen.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
+import React, {Component} from 'react';
+import {View} from 'react-native';
+
+import {globalStyles} from '../sheet/index';
+
+import NavigationHeader from '../components/NavigationHeader';
+import TextScrollView from '../components/TextScrollView';
+
+const tab = '\t\t';
```
Comment (ikalinin1) [inline src/screens/ShortScrollViewScreen.js:9]: 4 spaces instead of tab? :)

File: src/App.js (hunk 1 above)
Comment (uarlouski) [reply src/App.js:42]: it was formatted by linter due to line length

File: src/screens/ShortScrollViewScreen.js (hunk 2 above)
Comment (uarlouski) [reply src/screens/ShortScrollViewScreen.js:9]: it looks good visually

**Review by ikalinin1 — APPROVED** [review]

**Pull request merged**